clouddq/classes/dataplex_entity_schema.py

```diff
@@ -38,21 +42,43 @@ def from_dict(
             error_msg=f"Entity {entity_id}: must define non-empty value: 'fields'.",
         )
 
+        if kwargs.get("partitionFields"):
+            partition_fields = kwargs.get("partitionFields")
+            assert_not_none_or_empty(
+                value=partition_fields,
+                error_msg=f"Entity {entity_id}: must define non-empty value: 'partition_fields'.",
+            )
+        else:
+            partition_fields = None
+
+        if kwargs.get("partitionStyle"):
+            partition_style = kwargs.get("partitionStyle", None)
+            assert_not_none_or_empty(
+                value=partition_style,
+                error_msg=f"Entity {entity_id}: must define non-empty value: 'partition_style'."
+            )
+        else:
+            partition_style = None
+
         return DataplexEntitySchema(
             fields=fields,
```

Comment:
ditto here do you need to parse fields into DataplexEntitySchemaField?

Reply:
parsed to DataplexEntitySchemaField class